packages/pigment-css-unplugin/src/index.ts

```diff
@@ -302,17 +302,17 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
           }
 
           // Valid names must start with a underscore or letter.
-          const layerName = `_${slug}`;
-
-          // Fix for https://github.com/mui/pigment-css/issues/199
-          // Bring each file in its own layer so that the order is maintained between css modules
-          // shared between layout.tsx and page.tsx.
-          // TODO: Do this in a way that keeps the source map correct
-          cssText = `
-            @layer pigment.${layerName} {
-              ${cssText}
-            }
-          `;
+          // const layerName = `_${slug}`;
+
+          // // Fix for https://github.com/mui/pigment-css/issues/199
+          // // Bring each file in its own layer so that the order is maintained between css modules
+          // // shared between layout.tsx and page.tsx.
+          // // TODO: Do this in a way that keeps the source map correct
+          // cssText = `
```

Comment:
I don't think we are solving the same issue. We discussed with briefly with @Janpot, namely the layering for the base, overrides and variants styles should be there, but we should not remove this, because the conflict we had in the previous issue comes from defining two base styles (as we use the styled() call on a styled component):

```
// these are both styles that would be added in the base layer, no?
const Paper = styled(div)({});
const AppBar = styled(Paper)({});
```